wallets/config.js

```diff
@@ -63,7 +63,7 @@ export function useWalletConfigurator (wallet) {
         throw err
       }
     } else if (canReceive({ def: wallet.def, config: serverConfig })) {
-      const transformedConfig = await validateWallet(wallet.def, serverConfig)
+      const transformedConfig = await validateWallet(wallet.def, serverConfig, { skipGenerated: true })
```

Comment:
the client  needs to skip generated fields for receivers , because they are generated by the testCreateInvoice that runs serverside